CHANGELOG.md

```diff
@@ -140,6 +140,10 @@ information on these new classes, see our [Architecture Design] document.
 Again, we apologize for making this breaking change, but we believe it is in the
 best long-term interest of our customers.
 
+**BREAKING CHANGE:** The `bigtable::InstanceAdminClient` class has been marked
+as `final`. It should not be extended for mocking. Customers should instead use
```

Comment:
Maybe: s/It should not be extended for mocking/After the changes to this class in v1.3X.YY, we do not expect there is any need (or point) in extending this class./ ??

Reply:
I like the suggestion. Thanks.